ckanext/dalrrd_emc_dcpr/templates/dcpr/snippets/request_form.html

```diff
@@ -104,7 +105,27 @@
             information for aiding the NSIF and CSI in their review and moderation process.')) }}
         {% endcall %}
         {% if data.datasets %}
+            <legend>Dataset(s)</legend>
             {% for ds in data.datasets %}
+                {% set dataset_id = 'dataset'+ loop.index|string %}
+                <div class="panel panel-default" style="margin-bottom: 10px; display: block" id="{{ dataset_id }}">
+                 <div class="panel-heading">
+                    <h4 class="panel-title">
+                                {{ ds.proposed_dataset_title }}
+                            <button
+                                    type="button"
+                                    class="btn btn-danger btn-sm pull-right"
+                                    data-module="removeDcprRequestDatasets"
+                                    data-module-index="{{ loop.index }}"
+                                    style="margin-top: -2.5px;"
+                                     {% if data.datasets | count < 2 %}
```

Comment:
Hi, @ricardogsilva I think I added here the check the number of the dataset.  The button is disabled it is the only dataset

Reply:
It doesn't seem to work 100%, at least not on my local testing. I am able to go and edit an existing DCPR request, then delete all of its current datasets, then save and it works without any error.

Please double check on your side that it works OK when creating or updating DCPR requests and let me know. I will re-check after that

Reply:
Hi @ricardogsilva updated, thank you for the review
![Uploading remove_dcpr.gif…]()